### PR TITLE
chore: switch default S3 storage to MinIO

### PR DESCRIPTION
**Pull request by SdgJlbl**
fixes fl-1522

### CODE REVIEW

File: charts/substra-backend/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 <!-- towncrier release notes start -->
 
+## [24.6.0] - 2024-04-04
+
+### Changed
```
Comment (ThibaultFy) [inline charts/substra-backend/CHANGELOG.md:7]: I thought you added towncrier on this changelog too ?

**Review by ThibaultFy — APPROVED** [review]
lgtm :)

File: charts/substra-backend/CHANGELOG.md (hunk 1 above)
Comment (SdgJlbl) [reply charts/substra-backend/CHANGELOG.md:7]: Yes but for now the CI check hasn't been changed yet, and there is a question on whether it's useful to use towncrier for manual updates given that we bump the version number at every change.

**Pull request merged**